katello.gemspec

```diff
@@ -49,7 +49,7 @@ Gem::Specification.new do |gem|
 
   # Pulp
   gem.add_dependency "anemone"
-
+  gem.add_dependency "webrick"
```

Comment:
In running a test build on EL9/Ruby 3, (https://download.copr.fedorainfracloud.org/results/@theforeman/katello-nightly-staging-scratch-73818885-106f-5885-9ea9-0a7072eddbe2/rhel-9-x86_64/07004419-rubygem-katello/builder-live.log.gz), I saw this error and wondered if that is what is being fixed here?

```
+ /usr/bin/rake 'plugin:assets:precompile[katello]' RAILS_ENV=production DATABASE_URL=nulldb://nohost --trace
Log file /builddir/build/BUILD/katello-4.12.0.pre.master/usr/share/foreman/log/production.log cannot be opened. Falling back to STDOUT
Gem loading error: cannot load such file -- webrick/cookie
```

Reply:
I propose to get rid of anemone: https://projects.theforeman.org/issues/37159

Reply:
If we merge this, perhaps add a note above it why we need it?